ansible/dev.yml

```diff
@@ -0,0 +1,71 @@
+- name: setup local development environment
+  hosts: all
+
+  vars:
+    - spark_version: 1.3.1
+
+  tasks:
+    - name: install packages
+      sudo: yes
+      apt: name={{item}} state=installed update_cache=yes
+      with_items:
+        - libsnappy-dev
+        - liblzma-dev
+        - openjdk-7-jdk
+        - python-numpy
+        - python-pandas
```

Comment:
When we install this, the conda installation seems to go a lot faster. I think it's because the packages include binaries for things like blas and lapack. I'll add a comment.